File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/structure/autoincrement/TestModelAI2.java
package com.raizlabs.android.dbflow.test.structure.autoincrement;

import com.raizlabs.android.dbflow.annotation.Column;
import com.raizlabs.android.dbflow.annotation.ModelContainer;
import com.raizlabs.android.dbflow.annotation.PrimaryKey;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.structure.BaseModel;
import com.raizlabs.android.dbflow.test.TestDatabase;

@ModelContainer
@Table(database = TestDatabase.class)
public class TestModelAI2 extends BaseModel {

    @Column(name = "_id")
    @PrimaryKey(autoincrement = true)
    Long id;

    @Column
    String name;
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/structure/autoincrement/TestModelAI2.kt
package com.raizlabs.android.dbflow.test.structure.autoincrement

import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.PrimaryKey
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.structure.BaseModel
import com.raizlabs.android.dbflow.test.TestDatabase

@Table(database = TestDatabase::class)
class TestModelAI2 : BaseModel() {

    @Column(name = "_id")
    @PrimaryKey(autoincrement = true)
    var id: Long? = null

    @Column
    var name: String? = null
}
